seatunnel-connectors-v2/connector-assert/src/main/java/org/apache/seatunnel/connectors/seatunnel/assertion/sink/AssertSinkWriter.java

```diff
@@ -49,7 +54,18 @@ public void write(SeaTunnelRow element) {
 
     @Override
     public void close() throws IOException {
-        // nothing
+        assertRowRules.stream().filter(assertRule -> {
```

Comment:
I think It is easier to understand if we use `anyMatch` instead of `filter` and `findFirst` here.